doc/examples/ml/tensorflow/README.md

```diff
@@ -0,0 +1,59 @@
+# pix2pix with TensorFlow
+
+Christopher Hesse’s image-to-image demos use a Tensorflow implementation of the Generative Adversarial Networks (or GANs) model presented in this [article](https://arxiv.org/pdf/1611.07004v1.pdf).  Chris’s full Tensorflow implementation of this model can be found [on Github](https://github.com/affinelayer/pix2pix-tensorflow) and includes documentation about how to perform training, testing, pre-processing of images, exporting of the models for serving, and more.  
+
+In this example we will utilize Chris’s code in that repo along with a Docker image based on an image he created to run the scripts (which you can also utilize in your experiments with pix2pix).
+
+## The pipeline
+
+To deploy and manage the model discussed above, we will implement it’s training, export, preprocessing, and image generation in a reusable Pachyderm pipeline.  This will allow us to:
+
+- Keep a rigorous historical record of exactly what models were used on what data to produce which results.
+- Automatically update online ML models when training data or parameterization changes.
+- Easily revert to other versions of an ML model when a new model is not performing or when “bad data” is introduced into a training data set.
+
+The general structure of our pipeline will look like this:
+
+![Alt text](1.jpg)
+
+The cylinders represent data “repositories” in which Pachyderm will version training, model, etc. data (think “git for data”).  These data repositories are then input/output of the linked data processing stages (represented by the boxes in the figure).  
+
+## Getting up and running with Pachyderm
+
+You can experiment with this pipeline locally using a quick [local installation of Pachyderm](http://docs.pachyderm.io/en/v1.4.0-rc4/getting_started/local_installation.html).  Alternatively, you can quickly spin up a real Pachyderm cluster in any one of the popular cloud providers.  Check out the [Pachyderm docs](http://docs.pachyderm.io/en/v1.4.0-rc4/deployment/deploy_intro.html) for more details on deployment.
```

Comment:
fix[Pachyderm docs(http://docs.pachyderm.io/en/v1.4.0-rc4/deployment/deploy_intro.html) link? point to 1.4.0-rc4